mono/cil/opcode.mk

```diff
@@ -0,0 +1,6 @@
+# opcode.def is not built by default.
```

Comment:
This doesn't need to be in a different file, and has always worked as is. Remove this file and revert https://github.com/mono/mono/pull/9111/files#diff-e60505016a74c1c1b451e54c1d999a98L3 and https://github.com/mono/mono/pull/9111/files#diff-e60505016a74c1c1b451e54c1d999a98L13

Reply:
This is quite unclear to me -- the resulting dependency on timestamps, and the fact that CI does not have the tools installed to actually generate the file. CI failed earlier as a result. But let's try again. Maybe the timestamps were messed up before and can be reliably controlled.
It seems fragile to me, to ever depend on timestamps.